rfcs/0000-User&HostIntegrationRFC.md

```diff
@@ -0,0 +1,248 @@
+-->
+
+<!--
+Stage X: Provide a brief explanation of why the proposal is being marked as abandoned. This is useful context for anyone revisiting this proposal or considering similar changes later on.
+-->
+
+## Fields
+
+<!--
+Stage 1: Describe at a high level how this change affects fields. Include new or updated yml field definitions for all of the essential fields in this draft. While not exhaustive, the fields documented here should be comprehensive enough to deeply evaluate the technical considerations of this change. The goal here is to validate the technical details for all essential fields and to provide a basis for adding experimental field definitions to the schema. Use GitHub code blocks with yml syntax formatting, and add them to the corresponding RFC folder.
+-->
+
+### Proposed New Fields for User object
+
+Field | Type | Example | Description
+--- | --- | --- | ---
+user.profile.id	| keyword	| 1234	| User ID from the identity datasource.
+user.profile.type	| keyword	| Employee	| Type of user account.
+user.profile.status	| keyword |	On board	| Status of the user account.
+user.profile.first_name	| keyword |	First	| First Name of the User.
+user.profile.last_name	| keyword |	Last	| Last Name of the user.
+user.profile.other_identities	| keyword , multifield |	first.last@elk.elastic.co	| Array of additional user identities.
+user.profile.manager	| keyword |	John Doe	| Assigned Manager for the user account.
+user.profile.employee_type	| keyword |	Regular | Further classification type for the user account.
+user.profile.job_family	| keyword |	65-Sales	| Job family associated with the user account.
+user.profile.job_family_group	| keyword |	GTM	| Job family group associated with the user account.
+user.profile.management_level	| keyword |	Individual Contributor	| If the user account is identified as a Manager or Individual contributor.
+user.profile.job_title	| keyword |	Field Sales	| Job title assigned to the user account.
+user.profile.department	| keyword |	x256	| Department name associated with the user account.
+user.profile.organization	| keyword |	Elasticsearch Inc.	| Organization name associated with the account.
+user.profile.location	| keyword |	US - Washington - Distributed	| Assigned location for the user account.
+user.profile.mobile_phone	| keyword |	222-222-2222	
+user.profile.primaryPhone	| keyword |	222-222-2222	
+user.profile.secondEmail	| keyword |	first.l@elastic.co	| Additional email addresses associated with the user account.
+user.profile.sup_org_id	| keyword |	SUP-ORG-75	| Primary organization ID for the user account.
+user.profile.supervisory_Org	| keyword |	Field Sales	| Primary organization name for the user account.
+user.profile.assigned_mdm_id	| keyword |	2950	| Primary host owned by the user. This acts as a correlation to identify the host event.			
```

Comment:
So this correlates to an `asset.id` value? If so, I think the description should explicitly state this.

How do we qualify what is the "primary" device? For example, a user might be associated with a laptop, tablet, and phone.

Reply:
It can be the `asset.id` field value. It depends on the user's implementation of directory services and how they map their identifiers.

This ECS scope does not enforce or validate these correlation values. The data query logic will vary by the use case and is abstracted from this ECS definition.

I clarified the same in the field description- line 67.